internal/api/grpc/system/instance_converter.go

```diff
@@ -260,3 +261,22 @@ func fieldNameToInstanceDomainColumn(fieldName instance_pb.DomainFieldName) quer
 		return query.Column{}
 	}
 }
+
+func ListIAMMembersRequestToQuery(req *system_pb.ListIAMMembersRequest) (*query.IAMMembersQuery, error) {
+	offset, limit, asc := object.ListQueryToModel(req.Query)
+	queries, err := member_grpc.MemberQueriesToQuery(req.Queries)
+	if err != nil {
+		return nil, err
+	}
+	return &query.IAMMembersQuery{
+		MembersQuery: query.MembersQuery{
+			SearchRequest: query.SearchRequest{
+				Offset: offset,
+				Limit:  limit,
+				Asc:    asc,
+				// SortingColumn: model.IAMMemberSearchKey, //TOOD: not implemented in proto
```

Comment:
according to the PR template `All open todos and follow ups are defined in a new ticket and justified`... can you please link the issue here

Reply:
https://github.com/zitadel/zitadel/issues/5063